src/main.rs

```diff
@@ -845,29 +839,10 @@ fn cmd_delete(id: &str, state_dir: &str, matches: &ArgMatches) -> Result<()> {
     if let Ok(mut f) = File::open(PROCESS_PID) {
         let mut result = String::new();
         f.read_to_string(&mut result)?;
-        if let Ok(pid) = result.parse::<i32>() {
```

Comment:
This seems like a functional change. Was this intentional?

Reply:
Ouch! I'll fix that.